gravitee-am-identityprovider/gravitee-am-identityprovider-azure-ad/src/main/java/io/gravitee/am/identityprovider/azure/jwt/algo/Signature.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
I think this file can be removed in favor of this one : `io.gravitee.am.common.jwt.SignatureAlgorithm`